rest/route/middleware.go

```diff
@@ -815,6 +816,7 @@ func AddCORSHeaders(allowedOrigins []string, next http.HandlerFunc) http.Handler
 				w.Header().Add("Access-Control-Allow-Credentials", "true")
 				w.Header().Add("Access-Control-Allow-Methods", "POST, GET, OPTIONS, PATCH, PUT")
 				w.Header().Add("Access-Control-Allow-Headers", fmt.Sprintf("%s, %s, %s", evergreen.APIKeyHeader, evergreen.APIUserHeader, gqlHeader))
+				w.Header().Add("Access-Control-Max-Age", "600")
```

Comment:
I don't think we need to enforce that users can only make a request once every ten minutes. What's motivating this change? The UI uses this too so I think this will cause problems.

Reply:
This is only used by pre flight requests. Which are just used to identify if a CORS policy will be violated prior to making a request.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age

This will end up reducing load on evergreen (Although a very minuscule drop) . Since this will prevent 2 requests from being made for every POST request.